doc/10-icinga-template-library.md

```diff
@@ -3278,6 +3278,151 @@ printer_health_oids		| **Optional.** A list of oids which are downloaded and wri
 printer_health_offline		| **Optional.** The maximum number of seconds since the last update of cache file before it is considered too old.
 printer_health_multiline	| **Optional.** Multiline output
 
+#### Thola <a id="plugin-contrib-command-thola"></a>
+
+The [Thola](https://thola.io) plugin
+is a tool for monitoring network devices, that mainly uses SNMP.
+To monitor devices the subcommand `thola check` can be used.
+
+The `thola check` command has eight different subcommands:
+
+A documentation of these commands can be found by running `thola check -h`
+or in the plugin's API [swaggerhub documentation](https://app.swaggerhub.com/apis-docs/thola/thola/).
+To run these commands you need a server that is running the Thola API.
+If you don't know how to do this, you can also have a look at the plugin's
+[documentation](https://docs.thola.io). Also you have to
+put the Thola-client binary into the *PluginContribDir*.
+
+#### thola-check-cpu-load <a id="plugin-contrib-command-thola-check-cpu-load"></a>
```

Comment:
Such headings shall match the actual check command name as in the Icinga config.